ansible_collections/arista/avd/roles/eos_designs/python_modules/overlay/stun.py

```diff
@@ -26,14 +28,28 @@ def stun(self) -> dict | None:
         stun = {}
         if self.shared_utils.wan_role == "server":
             local_interfaces = []
-            # TODO - once the WAN interfaces are implemented, add them here
+            for wan_interface in self.shared_utils.wan_interfaces:
+                local_interfaces.append(wan_interface.get("interface"))
 
             stun["server"] = {"local_interfaces": local_interfaces}
 
         if self.shared_utils.wan_role == "client":
             server_profiles = []
 
-            # TODO - once the WAN interfaces are implemented, add them here
+            for wan_route_server, data in self._wan_route_servers.items():
```

Comment:
again overlap with _get_local_interfaces. Maybe a function filtering pathgroups on the ones where we need to peer with RR and add some extra meta data for the RR will make it simpler to parse everywhere.